x-pack/plugins/observability/typings/chart/index.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface ChartData {
+  time: number;
+  value: number;
+  group: string;
+}
```

Comment:
this is definitely not the final version. I just looked on the charts we are going to have and created a common schema for all of them, besides Metrics which has a different type of data. Looking forward to see what you'll propose here.

Reply:
Do you want to leave this up to a follow-up PR?